src/findBin.js

```diff
@@ -1,14 +1,22 @@
 'use strict'
 
 const parse = require('string-argv')
-const appRoot = require('app-root-path')
 const npmWhich = require('npm-which')(process.cwd())
 const checkPkgScripts = require('./checkPkgScripts')
 
-// Find and load the package.json at the root of the project.
-const pkg = require(appRoot.resolve('package.json')) // eslint-disable-line import/no-dynamic-require
 const debug = require('debug')('lint-staged:find-bin')
 
+// Find and load the package.json at the root of the project.
+let pkg
+try {
+  // eslint-disable-next-line import/no-dynamic-require, global-require
+  pkg = require(`${process.cwd()}/package.json`)
```

Comment:
What is going to happen if I run commit from a sub-directory? The idea was that `lint-staged` can still go up enough levels to locate the `package.json`. AFAIK this won't work anymore from sub-dirs.

Reply:
We want to look at the package.json responsible for the pre-commit hook right? That is the same as `process.cwd()`. In any case, like I mentioned, it won't break any valid config as we tolerate errors, if any, faced while loading it.

Reply:
> Replied with more info in https://github.com/okonet/lint-staged/pull/477#issuecomment-411339944

Commit from sub-directory works fine.